app/index.html

```diff
@@ -19,13 +19,6 @@
   </head>
   <body>
     {{content-for 'body'}}
-    <noscript>
-        <div id="main">
-            <div class='noscript'>
-                This site requires JavaScript to be enabled.
```

Comment:
Doing a final pass on these changes, I'm not sure if we should remove this notice at this time.  We'll want to make sure users hitting URLs other than `/policy` see this.  Is there a way to only remove this when rendering in fastboot?

Reply:
Good catch! Let me restore the warning for now. index.html cannot use Handlerbars apparently. So checking FastBoot from here seems tricky.